app/routes/companyInterest/components/CompanyInterestList.js

```diff
@@ -1,136 +1,142 @@
 // @flow
 import { ListNavigation } from 'app/routes/bdb/utils';
 import styles from './CompanyInterest.css';
-import React from 'react';
+import React, { Component } from 'react';
 import Button from 'app/components/Button';
 import { Link } from 'react-router';
-import Icon from 'app/components/Icon';
+import LoadingIndicator from 'app/components/LoadingIndicator';
 import { Content } from 'app/components/Content';
 import Flex from 'app/components/Layout/Flex';
 import type { CompanyInterestEntity } from 'app/reducers/companyInterest';
-import { ConfirmModalWithParent } from 'app/components/Modal/ConfirmModal';
+import Table from 'app/components/Table';
 
 export type Props = {
   companyInterestList: Array<CompanyInterestEntity>,
-  deleteCompanyInterest: number => Promise<*>
+  deleteCompanyInterest: number => Promise<*>,
+  fetch: Object => Promise<*>,
+  hasMore: boolean,
+  fetching: boolean
 };
 
-const fields = company => {
-  return [
-    {
-      value: company.companyName,
-      label: 'Bedriftsnavn'
-    },
-    {
-      value: company.contactPerson,
-      label: 'Kontaktperson'
-    },
-    {
-      value: company.mail,
-      label: 'Mail'
-    }
-  ];
+type State = {
+  clickedCompanyInterest: number
 };
 
-const CompanyInterestList = (props: Props) => {
-  const generateValues = company =>
-    fields(company).map(event => (
-      <td
-        key={`${event.label}-${company.id}`}
-        className={styles.companyInterestList}
-      >
-        <Link to={`/companyInterest/${company.id}/edit`}>{event.value}</Link>
-      </td>
-    ));
+const RenderCompanyActions = ({
+  id,
+  handleDelete,
+  fetching,
+  clickedCompanyInterest
+}: {
+  id: number,
+  handleDelete: number => void,
+  fetching: boolean,
+  clickedCompanyInterest: number
+}) => (
+  <div>
+    {fetching ? (
+      <LoadingIndicator loading={true} small />
```

Comment:
This loading indicator renders on all pages when fetching new company-interests, quite annoying.. Just a typo maybe?